src/algebra/hom/group.lean

```diff
@@ -1236,3 +1236,140 @@ protected lemma commute.map [mul_hom_class F M N] (h : commute x y) (f : F) :
 h.map f
 
 end commute
+
+section coe
+
+/-! ### Coercions as bundled morphisms
+
+The classes `coe_mul_hom`, `coe_monoid_hom`, etc. state that the coercion map `↑` (a.k.a. `coe`)
```

Comment:
Can you copy the explanation from the PR for why you use `has_lift_t` and not `has_coe` to this docstring?